docs/narr/viewconfig.rst

```diff
@@ -285,6 +285,23 @@ Non-Predicate Arguments
   are just developing stock Pyramid applications. Pay no attention to the man
   behind the curtain.
 
+``accept``
+  This value represents a match query for one or more mimetypes in the
```

Comment:
I'm not sure if "one or more mimetypes" was referring to the possibility of a media range in the argument representing multiple mimetypes (in which case that would no longer apply); or if it was referring to "one or more mimetypes in the header", in which case the match would not be only against mimetypes in the header, but also media ranges? Perhaps we could say something like "This is a media type, such as ``text/html``. If the ``Accept`` header is not present in the request or is invalid, or if the media type is considered acceptable by the ``Accept`` header, then this will match the current view..."?